sdk/template/Azure.Template/samples/Sample1.HelloWorldAsync.md

```diff
@@ -22,6 +22,3 @@ SecretBundle secret = await client.GetSecretValueAsync("TestSecret");
 
 Console.WriteLine(secret.Value);
 ```
-
-To see the full example source files, see:
```

Comment:
Do we know why this was included in the template in first place?  Curious why we authored this given the current concerns - what changed in our practices to change the guidance?

Reply:
When I first prototyped our new samples-as-tests with Pavel and Krzysztof there was some debate about how extracts may not be as useful as a full end-to-end sample. At one point we even discussed doing standalone apps much like Python does, but making sure we can test these - and, hence, relying on the MSTest framework (using NUnit to execute tests - was preferred.

It was a point in time thing, but after hearing customer feedback how they were confused by our samples (and turns out they were looking at these tests sources), we want to deemphasize the test sources as much as possible and lead people to the markdowns.